src/main/scala/is/hail/asm4s/FunctionBuilder.scala

```diff
@@ -90,6 +90,11 @@ class MethodBuilder(val fb: FunctionBuilder[_], mname: String, parameterTypeInfo
   def newLocal[T](name: String = null)(implicit tti: TypeInfo[T]): LocalRef[T] =
     new LocalRef[T](allocLocal[T](name))
 
+  def newField[T: TypeInfo]: ClassFieldRef[T] = newField()
+
+  def newField[T: TypeInfo](name: String = null): ClassFieldRef[T] =
+    new ClassFieldRef[T](fb, if (name == null) s"field${fb.cn.fields.size()}" else name)
```

Comment:
these two should just call `fb.newField`, right?

Reply:
oops, yes